src/components/organisms/AssetActions/Compute/index.tsx

```diff
@@ -190,6 +219,10 @@ export default function Compute({
   useEffect(() => {
     if (!ocean || !accountId) return
     checkPreviousOrders(ddo)
+    checkAssetDTMaxBuyQuantity(
+      price,
+      ddo.findServiceByType('metadata').attributes.main.type
+    )
   }, [ocean, ddo, accountId])
```

Comment:
shouldn't we include the price in the dependency array, so we recheck if the price changes?

Reply:
yep we can, at the moment we fetch the price only when the component mounts or ddo changes so `[ddo]` should be enough for the moment. It will be mandatory once we refetch the price from time to time

Reply:
ok, than it can be handled later